### DISCUSSION TITLE
Default password for user

### DISCUSSION THREAD

**Discussion by CraigCraig-jpeg** [actions/virtual-environments#6382] [General questions]
Whats the defautl password for Packer user?

**Comment by CraigCraig-jpeg** [comment]
Im trying to find the password for Packer to connect to the VM using puppet

**Comment by al-cheb** [comment]
https://www.packer.io/plugins/builders/azure/arm

[os_type](https://www.packer.io/plugins/builders/azure/arm#os_type) (string) - If either Linux or Windows is specified Packer will automatically configure authentication credentials for the provisioned machine. For Linux this configures an **SSH authorized key**. For Windows this configures a **WinRM certificate**.
